traefik/oam/register.go

```diff
@@ -117,3 +125,93 @@ func load(basePath string) ([]schemaDefinitionPathSet, error) {
 
 	return res, nil
 }
+
+//RegisterWorkLoadsDynamically ...
+func RegisterWorkLoadsDynamically(runtime, host string) error {
+	av, cv, err := getLatestValidAppVersionAndChartVersion()
+	if err != nil {
+		return err
+	}
+	fmt.Println("version ", av)
+	m := manifests.Config{
+		Name:        "Traefik",
+		MeshVersion: av,
+		Filter: manifests.CrdFilter{
+			RootFilter:    []string{"$[?(@.kind==\"CustomResourceDefinition\")]"},
+			NameFilter:    []string{"$..[\"spec\"][\"names\"][\"kind\"]"},
+			VersionFilter: []string{"$..spec.versions[0]", " --o-filter", "$[0]"},
+			GroupFilter:   []string{"$..spec", " --o-filter", "$[]"},
+			SpecFilter:    []string{"$..openAPIV3Schema.properties.spec", " --o-filter", "$[]"},
+		},
+	}
+	url := "https://helm.traefik.io/mesh/maesh-" + cv + ".tgz"
```

Comment:
We're going to want this to be an externally configurable location. If we implement the `RegisterWorkLoadsDynamically` function in MeshKit or the Adapter Library, it should include the ability to pass in this location. // @DelusionalOptimist @utkarsh-pro, correct?